sdk/loadtestservice/load-testing-rest/README.md

```diff
@@ -0,0 +1,192 @@
+
+### Test Engine
+
+A test engine is computing infrastructure that runs the Apache JMeter test script. You can scale out your load test by configuring the number of test engines. The test script runs in parallel across the specified number of test engines.
+
+### Test Run
+
+A test run represents one execution of a load test. It collects the logs associated with running the Apache JMeter script, the load test YAML configuration, the list of app components to monitor, and the results of the test.
+
+### Data-Plane Endpoint
+
+Data-plane of Azure Load Testing resources is addressable using the following URL format:
+
+`00000000-0000-0000-0000-000000000000.aaa.cnt-prod.loadtesting.azure.com`
+
+The first GUID `00000000-0000-0000-0000-000000000000` is the unique identifier used for accessing the Azure Load Testing resource. This is followed by  `aaa` which is the Azure region of the resource.
+
+The data-plane endpoint is obtained from Control Plane APIs.
+
+**Example:** `1234abcd-12ab-12ab-12ab-123456abcdef.eus.cnt-prod.loadtesting.azure.com`
+
+In the above example, `eus` represents the Azure region `East US`.
+
+## Examples
+
+### Creating a load test
+
+```javascript
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+
+var TEST_ID = "some-test-id";
+var DISPLAY_NAME = "my-load-test";
+
+const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
+
+await Client.path("/loadtests/{testId}", TEST_ID).patch({
+  contentType: "application/merge-patch+json",
+  body: {
+    displayName: DISPLAY_NAME,
+    description: "",
+    loadTestConfig: {
+      engineInstances: 1,
+      splitAllCSVs: false,
+    },
+  },
+});
+```
+
+### Uploading .jmx file to a Test
+
+```javascript
+import { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+import { createReadStream } from "fs";
+
+var TEST_ID = "some-test-id";
+var FILE_ID = "some-file-id";
+const readStream = createReadStream("./sample.jmx");
+
+const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
+
+await Client.path("/loadtests/{testId}/files/{fileId}", TEST_ID, FILE_ID).put({
+  contentType: "multipart/form-data",
+  body: {
+    file: readStream,
+  },
+});
+```
+
+### Running a Test
+
+```javascript
+import { AzureLoadTestingClient } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+
+var TEST_ID = "some-test-id";
+var TEST_RUN_ID = "some-testrun-id";
+var DISPLAY_NAME = "my-load-test-run";
+
+const Client: AzureLoadTestingClient = AzureLoadTesting(Endpoint, new DefaultAzureCredential());
+
+await Client.path("/testruns/{testRunId}", "abcde").patch({
+  contentType: "application/merge-patch+json",
+  body: {
+    testId: TEST_ID,
+    displayName: DISPLAY_NAME,
+  },
+});
+```
+
```

Comment:
Add the code for fetching client metrics for the above test run to complete the end-to-end scenario.

Reply:
Updated.